daemon/monitor.go

```diff
@@ -85,16 +85,29 @@ func (daemon *Daemon) StateChanged(id string, e libcontainerd.StateInfo) error {
 		}
 		return daemon.postRunProcessing(c, e)
 	case libcontainerd.StateExitProcess:
+		closeAsync := make(chan bool)
```

Comment:
The value from the channel is never used, I'd rather have `chan struct{}` in that case.